### PR TITLE
Update 2fa to include troubleshooting

### PR DESCRIPTION
**Pull request by kayla-broussard**
Two troubleshooting tips mentioned - ensuring your phone's time is appropriately synchronized, and waiting for a token refresh.

Closes #444

### CODE REVIEW

File: content/articles/two-factor-authentication.markdown [hunk 1]
```diff
@@ -134,3 +134,23 @@ There lots of time-based two-factor authentication apps that generate verificati
...
+## Troubleshooting two-factor authentication {#troubleshooting}
+
+
+##### Ensuring your phone's date and time synchronized
+
+Authenticator app two-factor authentication codes are based on time and thus require the time on your phone to be in-sync with the official time. If your phone's time is out of sync, or if you have changed time zones, the best way to fix it is to change the time settings on your phone from "Manual" to "Automatic".
+
+- [This article from Apple's help centre will provide steps for iOS](https://support.apple.com/en-ie/HT203483).
+- Changing this setting on Android may vary, but a [general guide can be found here](https://squareup.com/help/us/en/article/5288-update-date-time-on-your-device).
+-  You can also try the [Time Sync feature if you use Google Authenticator for Android](https://support.google.com/accounts/answer/2653433?visit_id=636735803516007086-2349741641&p=timesync&rd=1).
```
Comment (sbastn) [inline content/articles/two-factor-authentication.markdown]: You can remove the `visit_id` param from this URL so it is just `https://support.google.com/accounts/answer/2653433`

Comment (kayla-broussard) [reply content/articles/two-factor-authentication.markdown]: Thanks Seb, changed :)

**Comment by sbastn** [comment]
Is there anything else preventing this from being merged?

**Comment by kayla-broussard** [comment]
> 
> 
> Is there anything else preventing this from being merged?

I was waiting on an approval from @weppos as well as he's still pending, but am happy to otherwise merge if that's not an issue.

File: content/articles/two-factor-authentication.markdown [hunk 2]
```diff
@@ -134,3 +134,24 @@ There lots of time-based two-factor authentication apps that generate verificati
...
+
+#### Ensuring your phone's date and time synchronized
+
+Two-factor authentication codes are based on time and thus require the time on your phone to be in-sync with the official time. Otherwise, your authenticator app will generate a mismatching code. If your phone's time is out of sync, or if you have changed time zones, the best way to fix it is to change the time settings on your phone from "Manual" to "Automatic".
+
+- [This article from Apple's help centre will provide steps for iOS](https://support.apple.com/en-ie/HT203483).
+- Changing this setting on Android may vary, but a [general guide can be found here](https://squareup.com/help/us/en/article/5288-update-date-time-on-your-device).
+- You can also try the [Time Sync feature if you use Google Authenticator for Android](https://support.google.com/accounts/answer/2653433).
+
+#### Generate a fresh authentication code
```
Comment (weppos) [inline content/articles/two-factor-authentication.markdown]: I believe we should use a tone for this time that is more trouble-shooting like (also for consistency). Such as "Using a freshly-generated authentication code".

Comment (itsalyse) [reply content/articles/two-factor-authentication.markdown]: Agree.

**Pull request merged**